tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckCoverage.java

```diff
@@ -131,9 +131,9 @@ public CheckCLDR handleSetCldrFileToCheck(
             supplementalData =
                     SupplementalDataInfo.getInstance(cldrFileToCheck.getSupplementalDirectory());
             coverageLevel = CoverageLevel2.getInstance(supplementalData, localeID);
-            PluralInfo pluralInfo = supplementalData.getPlurals(PluralType.cardinal, localeID);
-            if (pluralInfo == supplementalData.getPlurals(PluralType.cardinal, LocaleNames.ROOT)
-                    && !SpecialLocales.isScratchLocale(localeID)) {
+            PluralInfo pluralInfo =
+                    supplementalData.getPlurals(PluralType.cardinal, localeID, false);
+            if (pluralInfo == null && !SpecialLocales.isScratchLocale(localeID)) {
```

Comment:
Previously, the warning would always fire for Japanese, etc, because the plural rules matched root. This changes to only fire if plural rules were not found.